db-service/test/cds-infer/elements.test.js

```diff
@@ -201,6 +201,7 @@ describe('infer elements', () => {
       let { Books, Receipt } = model.entities
       expect(inferred).to.have.nested.property('sources.Books', Books)
       expect(inferred).to.have.nested.property('sources.Receipt', Receipt)
+      // eslint-disable-next-line no-unused-vars
       const { image, ...BooksElementsWithoutBlob } = Books.elements
```

Comment:
```suggestion
      const { image: _image, ...BooksElementsWithoutBlob } = Books.elements
```
That should also work.

Reply:
Reason: ESLint allows unused variables if they start with an underscore.

Reply:
Thanks David!